Cargo.toml

```diff
@@ -131,10 +133,15 @@ alloy-consensus = { version = "1.0.18", default-features = false }
 alloy-primitives = { version = "1.2.0", default-features = false }
 alloy-hardforks = { version = "0.2.7", default-features = false }
 alloy-chains = { version = "0.2.5", default-features = false }
+alloy-rlp = { version = "0.3.10", default-features = false }
+alloy-trie = { version = "0.9.0", default-features = false }
 
 ethereum_ssz_derive = "0.9"
 ethereum_ssz = "0.9"
 
+revm-bytecode = { version = "6.0.1", default-features = false }
+
+risc0-ethereum-trie = { git = "https://github.com/risc0/risc0-ethereum.git", rev = "4a5efe13b3655258b89e7082c57743e445133dc8" }
```

Comment:
Sticking to the [same reference](https://github.com/risc0/zeth/blob/main/Cargo.toml#L20)

Reply:
Can we create a tracking issue to eventually remove this (the MPT impl) when/if it gets added to upstream?